cmd/util/client.go

```diff
@@ -0,0 +1,12 @@
+package cmdutil
+
+import (
+	"github.com/puppetlabs/wash/api/client"
+	"github.com/puppetlabs/wash/cmd/internal/config"
+)
+
+// NewClient returns a new Wash client for the given subcommand.
+// Tests can set NewClient to a stub that returns a mock client.
+var NewClient = func() client.Client {
```

Comment:
Once we move the `api/fs` stuff over to the client, this can become `NewClient(withLocalFSSupport bool)`.